mindsdb_sdk/chatbots.py

```diff
@@ -0,0 +1,176 @@
+from typing import List
+from mindsdb_sdk.utils.objects_collection import CollectionBase
+
+
+class Chatbot:
+    """
+    Represents a chatbot that can be managed within a project.
+    """
+
+    def __init__(self, api, project, data: dict):
+        self.api = api
+        self.project = project
+        self.name = data.get('name')
+        self.database_name = data.get('database')
+        self.agent_name = data.get('agent')
+        self.model_name = data.get('model_name')
+        self.is_running = data.get('is_running')
+
+    def __repr__(self):
+        return f"{self.__class__.__name__}({self.project.name}.{self.name})"
+
+    def update(self, name: str = None, agent_name: str = None, model_name: str = None, database_name: str = None, inplace: bool = False):
+        """
+        Updates the chatbot's properties.
+
+        Example usage:
+            >>> chatbot.update(model_name='gpt-4', database_name='slack_db')
+
+        :param name: (Optional) New name for the chatbot.
+        :param agent_name: (Optional) New agent name to associate with the chatbot.
+        :param model_name: (Optional) New model to use for the chatbot.
+        :param database_name: (Optional) New database connection name.
+        :param inplace: If True, updates the current object in-place; otherwise, returns a new Chatbot object.
+        :return: Updated Chatbot object, or None if inplace is True.
+        """
+        payload = {}
+
+        if name:
+            payload['name'] = name
+
+        if database_name:
+            payload['database_name'] = database_name
+
+        if agent_name:
+            payload['agent_name'] = agent_name
+
+        if model_name:
+            payload['model_name'] = model_name
+
+        updated_chatbot = self.api.update_chatbot(
+            project=self.project.name,
+            chatbot_name=self.name,
+            data=payload
+        )
+
+        if inplace:
```

Comment:
What could be the reason to update chatbot on server and not update local instance?

Reply:
I think there's no reason for it 😅😅